root/meta/race-12552/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+ROOTTEST_GENERATE_REFLEX_DICTIONARY(libUser User.hpp SELECTION User.xml LIBNAME libUser NO_ROOTMAP)
+set(depends ${GENERATE_REFLEX_TEST})
+
+if(NOT MSVC)
+   ROOTTEST_ADD_AUTOMACROS(DEPENDS ${depends})
```

Comment:
If the test is to be disabled on Windows, I think `ROOTTEST_GENERATE_REFLEX_DICTIONARY()` in line 1 can also be skipped, right?  That should save some cycles.